crosscorrelation.Rmd

```diff
@@ -0,0 +1,384 @@
+---
+title: "uploadid"
+author: "Amenze"
+date: "June 28, 2018"
+output: html_document
+---
+
+```{r}
+setwd("/Users/Amenze/Desktop/tidepool/refdata")
```

Comment:
Remove this since there's no guarantee anyone else will have the same directory structure.